cristin-commons/src/main/java/no/unit/nva/cristin/common/handler/CristinQueryHandler.java

```diff
@@ -41,31 +42,38 @@ protected String getValidPage(RequestInfo requestInfo) throws BadRequestExceptio
         if (Utils.isPositiveInteger(page)) {
             return page;
         }
-        throw new BadRequestException(ERROR_MESSAGE_PAGE_VALUE_INVALID);
+        throw new BadRequestException(String.format(ERROR_MESSAGE_INVALID_VALUE, PAGE));
     }
 
     protected String getValidNumberOfResults(RequestInfo requestInfo) throws BadRequestException {
         String results = getQueryParameter(requestInfo, NUMBER_OF_RESULTS).orElse(DEFAULT_NUMBER_OF_RESULTS);
         if (Utils.isPositiveInteger(results)) {
             return results;
         }
-        throw new BadRequestException(ERROR_MESSAGE_NUMBER_OF_RESULTS_VALUE_INVALID);
+        throw new BadRequestException(String.format(ERROR_MESSAGE_INVALID_VALUE, NUMBER_OF_RESULTS));
     }
 
     protected String getValidQuery(RequestInfo requestInfo) throws BadRequestException {
         return getQueryParameter(requestInfo, QUERY)
                 .filter(this::isValidQueryString)
                 .map(UriUtils::escapeWhiteSpace)
-                .orElseThrow(() -> new BadRequestException(ERROR_MESSAGE_QUERY_MISSING_OR_HAS_ILLEGAL_CHARACTERS));
+                .orElseThrow(() -> new BadRequestException(invalidQueryParametersMessage(
+                        QUERY, ALPHANUMERIC_CHARACTERS_DASH_COMMA_PERIOD_AND_WHITESPACE)));
     }
 
     protected String getValidName(RequestInfo requestInfo) throws BadRequestException {
         return getQueryParameter(requestInfo, NAME)
                 .filter(this::isValidQueryString)
                 .map(UriUtils::escapeWhiteSpace)
-                .orElseThrow(() -> new BadRequestException(ERROR_MESSAGE_NAME_MISSING_OR_HAS_ILLEGAL_CHARACTERS));
+                .orElseThrow(() -> new BadRequestException(
+                        invalidQueryParametersMessage(NAME, ALPHANUMERIC_CHARACTERS_DASH_COMMA_PERIOD_AND_WHITESPACE)));
     }
 
+    protected Optional<String> getValidOrganization(RequestInfo requestInfo) {
+        return getQueryParameter(requestInfo, ORGANIZATION)
+                .filter(this::isValidQueryString)
+                .map(UriUtils::escapeWhiteSpace);
```

Comment:
Do this escapeWhiteSpace change "Norwegian Business School" to "NorwegianBusinessSchool"?